Dockerfile-alpine

```diff
@@ -1,5 +1,5 @@
 FROM postgres:12.2-alpine
```

Comment:
Not 🆒

Reply:
```suggestion
FROM postgres:12.3-alpine
```

Reply:
I should mention this in https://github.com/citusdata/packaging/issues/500